.github/workflows/ci.yaml

```
@@ -111,15 +152,15 @@ jobs:
 
       - name: Run tests
 
-        run: poetry run pytest -m "not agentjobs" -W default::great_expectations.datasource.fluent.GxInvalidDatasourceWarning --cov-report xml
+        run: poetry run pytest -m integration -W default::great_expectations.datasource.fluent.GxInvalidDatasourceWarning --cov-report xml
```

```suggestion
        run: poetry run pytest -vv -m integration -W default::great_expectations.datasource.fluent.GxInvalidDatasourceWarning --cov-report xml
```
We shouldn't have that many integration tests so I think using the `verbose` flag can be helpful especially when these tests get slower.